sbt-plugin-test/build.sbt

```diff
@@ -11,7 +11,10 @@ val versionSettings = Seq(
 
 val baseSettings = versionSettings ++ Seq(
   libraryDependencies +=
-    "org.scala-js" %% "scalajs-jasmine-test-framework" % scalaJSVersion % "test"
+    "org.scala-js" %% "scalajs-jasmine-test-framework" % scalaJSVersion % "test",
+
+  // Test that non-existent classpath entries are allowed - #2198
+  fullClasspath += file("./non-existent-directory-please-dont-ever-create-this")
```

Comment:
Maybe, but by accident. 

The only place where sbt guarantees the meaning of `.` is in the initial setting of projects' base directories as in `.in(file("."))`. Elsewhere it's undefined.